doc/ref/cli/salt-ssh.rst

```diff
@@ -53,38 +57,117 @@ Options
     the more running process the faster communication should be, default
     is 25.
 
+.. option:: --extra-filerefs=EXTRA_FILEREFS
+
+   Pass in extra files to include in the state tarball.
+
+.. option:: --min-extra-modules=MIN_EXTRA_MODS
+
+    One or comma-separated list of extra Python modulesto be included
+    into Minimal Salt.
+
+.. option:: --thin-extra-modules=THIN_EXTRA_MODS
+
+   One or comma-separated list of extra Python modulesto  be included
+   into Thin Salt.
+
+.. option:: -v, --verbose
+
+   Turn on command verbosity, display jid.
+
+.. option:: -s, --static
+
+   Return the data from minions as a group after they all
```

Comment:
This description isn't quite complete. Maybe missing `return` at the end of the sentence?